tests/e2e/scripts/keadm_e2e.sh

```diff
@@ -115,6 +124,8 @@ prepare_cluster
 echo -e "\nBuilding cloud image..."
 build_image
 
+install_cni_plugins
```

Comment:
Now if user uses `keadm` to install edgecore, it's need to install the CNI first? cc @gy95 @Shelley-BaoYue

Reply:
yes, ref: https://github.com/containerd/containerd/blob/main/docs/getting-started.md

Reply:
Okey, let's also add some docs for joining node when user use `containerd`, can notes user to to follow the link above to install CNI and other necessary components @gy95

Reply:
yes, already has a doc https://kubeedge.io/en/docs/advanced/cri/
we can update it to the latest

Reply:
Okey, the docs above is for setting different runtimes, we can also add some notes or link for swithing to  `containerd` in `join node` part :)